src/Symfony/Component/HttpFoundation/Tests/Session/Storage/Proxy/SessionHandlerProxyTest.php

```diff
@@ -28,128 +27,159 @@
+        $handler->expects($this->once())
             ->method('close')
             ->willReturn(false);
 
-        $this->assertFalse($this->proxy->isActive());
-        $this->proxy->close();
-        $this->assertFalse($this->proxy->isActive());
+        $this->assertFalse($proxy->isActive());
+        $proxy->close();
+        $this->assertFalse($proxy->isActive());
     }
 
     public function testRead()
     {
-        $this->mock->expects($this->once())
+        $handler = $this->createMock(\SessionHandlerInterface::class);
+        $proxy = new SessionHandlerProxy($handler);
+        $handler->expects($this->once())
             ->method('read')
             ->willReturn('foo')
         ;
 
-        $this->proxy->read('id');
+        $proxy->read('id');
     }
 
     public function testWrite()
     {
-        $this->mock->expects($this->once())
+        $handler = $this->createMock(\SessionHandlerInterface::class);
+        $proxy = new SessionHandlerProxy($handler);
+        $handler->expects($this->once())
             ->method('write')
             ->willReturn(true)
         ;
 
-        $this->assertTrue($this->proxy->write('id', 'data'));
+        $this->assertTrue($proxy->write('id', 'data'));
     }
 
     public function testDestroy()
     {
-        $this->mock->expects($this->once())
+        $handler = $this->createMock(\SessionHandlerInterface::class);
+        $proxy = new SessionHandlerProxy($handler);
+        $handler->expects($this->once())
             ->method('destroy')
             ->willReturn(true)
         ;
 
-        $this->assertTrue($this->proxy->destroy('id'));
+        $this->assertTrue($proxy->destroy('id'));
     }
 
     public function testGc()
     {
-        $this->mock->expects($this->once())
+        $handler = $this->createMock(\SessionHandlerInterface::class);
+        $proxy = new SessionHandlerProxy($handler);
+        $handler->expects($this->once())
             ->method('gc')
             ->willReturn(1)
         ;
 
-        $this->proxy->gc(86400);
+        $proxy->gc(86400);
+    }
+
+    public function testValidateIdWithoutUpdateTimestampHandler()
+    {
+        $proxy = new SessionHandlerProxy($this->createStub(\SessionHandlerInterface::class));
+
+        $this->assertTrue($proxy->validateId('id'));
     }
 
-    public function testValidateId()
+    public function testValidateIdWithUpdateTimestampHandlerAndValidId()
     {
-        $mock = $this->createMock(TestSessionHandler::class);
-        $mock->expects($this->once())
-            ->method('validateId');
+        $handler = $this->createMock(TestSessionHandler::class);
+        $handler->expects($this->once())
+            ->method('validateId')
+            ->willReturn(true);
 
-        $proxy = new SessionHandlerProxy($mock);
-        $proxy->validateId('id');
+        $proxy = new SessionHandlerProxy($handler);
 
-        $this->assertTrue($this->proxy->validateId('id'));
```

Comment:
This doesn't seem to have made much sense before as the mock we configured before was never used here.